README.md

```diff
@@ -143,6 +143,44 @@ guard condition else { return }
 if condition { } else { }
 ```
 
+### Commas
+
+There should be no space before an comma and a single one after any comma.
+
+**Preferred:**
+```swift
+function print(name: String, surname: String) {
+  ...
+}
+```
+
+**Not Preferred:**
+```swift
+function print(name: String ,surname: String) {
+  ...
+}
+```
+
+## Parentheses
+
+Parentheses around conditionals, if,for,while,do statements, are not required and should be omitted.
+
+**Preferred:**
+```swift
+if name == "Hello" {
+  print("World")
+}
+```
+
+**Not Preferred:**
+```swift
+if (name == "Hello") {
+  print("World")
+}
+```
+
+In larger expressions, optional parentheses can sometimes make code read more clearly.
```

Comment:
Do you have any examples of this in the code? I can't think of a good exception and I'd rather remove this comment

Reply:
I'm not sure I agree with the wording, but I agree with the idea we should aim for no optional parentheses.

This is an exaggeration, but it shows there's at least some truth in the statement above:

<img width="230" alt="screen shot 2017-10-16 at 1 41 19 pm" src="https://user-images.githubusercontent.com/1836005/31624049-c103debc-b277-11e7-928d-67a24c53348b.png">